centreon/src/Core/Security/Authentication/Domain/Provider/OpenIdProvider.php

```diff
@@ -189,6 +189,7 @@ public function createUser(): void
             $this->userInformations[$customConfiguration->getUserNameBindAttribute()],
             $this->userInformations[$customConfiguration->getEmailBindAttribute()],
         );
+        ! $user->canReachFrontend() ?: $user->setCanReachRealtimeApi(true);
```

Comment:
Elvis operator with negation operator is a little hard to understand but i see what you want to do, 

Just a suggestion for readability 

```
if ($user->canReachFrontend()) {
    $user->setCanReachRealtimeApi(true);
}

Reply:
Done